test/test_ops.py

```diff
@@ -1340,10 +1341,65 @@ def is_bit_set(x):
             torch.is_complex,
         )
 
+# A simple wrapper tensor subclass that runs runs
+# correctness checks to ensure that operators have expected
+# tags based on their input and ouput tensor properties
+class WrapperToTestTags(torch.Tensor):
```

Comment:
A mode seems like a better way to implement this

Reply:
agreed! updated